src/tunacode/ui/commands/resume.py

```
@@ -0,0 +1,143 @@
+            "list": self._handle_list,
+            "load": self._handle_load,
+            "delete": self._handle_delete,
+        }.get(subcommand)
+
+        if handler is None:
+            app.notify(f"Unknown subcommand: {subcommand}", severity="warning")
+            return
+
+        await handler(app, parts)
+
+    async def _handle_list(self, app: TextualReplApp, parts: list[str]) -> None:
```

Ruff (0.15.0)

[warning] Unused method argument: `parts`

(ARG002)